transport/stream.go

```diff
@@ -96,7 +96,8 @@ func (e *TCPEndpoint) Connect(ctx context.Context) (StreamConn, error) {
 	return conn.(*net.TCPConn), nil
 }
 
-// StreamDialerEndpoint is a [StreamEndpoint] that connects to the given address using the given [StreamDialer].
+// StreamDialerEndpoint is a [StreamEndpoint] that connects to the specified address using the specified
```

Comment:
[StreamDialerEndpoint]

Reply:
Type names should not be wrapped in `[]` in the documentation that describes the type itself.